test/createNewProject.test.ts

```diff
@@ -66,7 +66,7 @@ suite('Create New Project Tests', async function (this: ISuiteCallbackContext):
         await validateVSCodeProjectFiles(projectPath);
         const projectName: string = path.basename(projectPath);
         assert.equal(await fse.pathExists(path.join(projectPath, `${projectName}.csproj`)), true, 'csproj does not exist');
-        await validateSetting(projectPath, `${extensionPrefix}.${deploySubpathSetting}`, 'bin/Release/netstandard2.0/publish');
+        await validateSetting(projectPath, `${extensionPrefix}.${deploySubpathSetting}`, 'bin/Release/netcoreapp2.1/publish');
```

Comment:
Should we validate what version of .NET they're running before setting their deployPath?  This could a tricky problem for users to fix if they're unaware of the setting.

Reply:
Never mind, this is a test file.